core-web/libs/sdk/react/tsconfig.lib.json

```diff
@@ -19,5 +19,12 @@
         "src/**/*.spec.jsx",
         "src/**/*.test.jsx"
     ],
-    "include": ["src/**/*.js", "src/**/*.jsx", "src/**/*.ts", "src/**/*.tsx"]
+    "include": [
```

Comment:
Isn't this redundant giving this `"extends": "./tsconfig.json"`?

Reply:
Done!